src/neptune/internal/operation_processors/async_operation_processor.py

```diff
@@ -84,10 +84,10 @@ def __init__(
     @staticmethod
     def _init_data_path(container_id: UniqueId, container_type: ContainerType) -> Path:
         now = datetime.now()
-        container_dir = f"{NEPTUNE_DATA_DIRECTORY}/{ASYNC_DIRECTORY}/{container_type.create_dir_name(container_id)}"
-        data_path = f"{container_dir}/exec-{now.timestamp()}-{now.strftime('%Y-%m-%d_%H.%M.%S.%f')}-{os.getpid()}"
-        data_path = data_path.replace(" ", "_").replace(":", ".")
-        return Path(data_path)
+        process_path = f"exec-{now.timestamp()}-{now.strftime('%Y-%m-%d_%H.%M.%S.%f')}-{os.getpid()}".replace(
```

Comment:
We probably don't need `.replace(" ", "_")` anymore and it's some kind of a leftover. What do you think?